blenderAddons/vispCAOExport/setup.py

```diff
@@ -0,0 +1,9 @@
+import bpy
+import sys
+import os
+
+print("BLENDER VERSION : " + bpy.app.version_string)
+ADDON_PATH = (bpy.utils.user_resource('SCRIPTS', "addons"))
+
+os.system("cp -r ../vispCAOExport " + ADDON_PATH)
+print("Moved addon to " + ADDON_PATH)
```

Comment:
There actually may be a need for a script to update the plugin since on startup, blender will load the plugin from the .config/ directory, which is not a git repo, unless the user clones this project there.
I'll see if there already exist scripts to handle this.